db/dbformat.h

```diff
@@ -327,21 +327,42 @@ inline int InternalKeyComparator::Compare(const InternalKey& a,
   return Compare(a.Encode(), b.Encode());
 }
 
+inline Status ReturnCorruptKeyInfo(ParsedInternalKey* ikey, bool log_err_key) {
```

Comment:
Can we remove `inline`? Since it's an exception case, we probably don't want to enlarge the binary for the function for this case.